insights/ui/login.py

```diff
@@ -0,0 +1,34 @@
+import configparser
+from selenium_utility import SeleniumUtility
+from locators import LoginPageLocator, InsightPageLocator
```

Comment:
if you add all the elements to one dict, then you will not have to write multiple imports like this.